src/views/Dashboard/Customer/Home/index.vue

```diff
@@ -28,24 +28,335 @@
+  getOrdersData
+} from "@/common/lib/polkadot-provider/query/orders"
+import { queryLabsById } from "@/common/lib/polkadot-provider/query/labs"
+import { queryServicesById } from "@/common/lib/polkadot-provider/query/services"
 
 export default {
   name: "CustomerHome",
 
-  components: {Banner },
-  data: () => ({ creditCardIcon, layersIcon, labIllustration, cardBlock: false }),
+  components: { Banner, DataTable, Button },
+  data: () => ({
+    creditCardIcon,
+    layersIcon,
+    labIllustration,
+    cardBlock: false,
+    eyeIcon,
+    orderHistory: [],
+    isLoadingOrderHistory: false,
+    headers: [
+      { text: "Service Name", value: "title" },
+      { text: "Lab Name", value: "labName" },
+      { text: "Date", value: "orderDate" },
+      { text: "Status", value: "status" },
+      {
+        text: "Actions",
+        value: "actions",
+        sortable: false,
+        align: "center",
+        width: "5%"
+      }
+    ],
+    dummyItems: [
+      {
+        dna_sample_tracking_id: "AYEY6073POOH",
+        icon: "mdi-dna",
+        labName: "GSI Lab",
+        number: "0x3943c10e33521319e4222843c33a939492978da07070c15815cc32703712604a",
+        orderDate: "20 Jan 2021",
+        status: "Fulfilled",
+        timestamp: "1631849952000",
+        title: "Covid-19 Testing"
+      },
+      {
+        dna_sample_tracking_id: "073PAYEYH6OO",
+        icon: "mdi-dna",
+        labName: "Nebula Lab",
+        number: "0x3943c10e33521319e4222843c33a939492978da07070c15815cc32703712604a",
+        orderDate: "20 Jan 2021",
+        status: "Paid",
+        timestamp: "1631849952000",
+        title: "Whole Genome Sequencing"
+      },
+      {
+        dna_sample_tracking_id: "073PY6OHOAYE",
+        icon: "mdi-dna",
+        labName: "Prodia",
+        number: "0x3943c10e33521319e4222843c33a939492978da07070c15815cc32703712604a",
+        orderDate: "20 Jan 2021",
+        status: "Unpaid",
+        timestamp: "1631849952000",
+        title: "Whole Exome Sequencing"
+      },
+      {
+        dna_sample_tracking_id: "6073POAHYEYO",
+        icon: "mdi-dna",
+        labName: "Speed Lab",
+        number: "0x3943c10e33521319e4222843c33a939492978da07070c15815cc32703712604a",
+        orderDate: "20 Jan 2021",
+        status: "Refunded",
+        timestamp: "1631849952000",
+        title: "Targeted Gene Panel Sequencing"
+      },
+      {
+        dna_sample_tracking_id: "EYO6073POHAY",
+        icon: "mdi-dna",
+        labName: "32andme",
+        number: "0x3943c10e33521319e4222843c33a939492978da07070c15815cc32703712604a",
+        orderDate: "20 Jan 2021",
+        status: "Canceled",
+        timestamp: "1631849952000",
+        title: "SNP Microarray"
+      }
+    ]
+  }),
 
   mounted() {
     window.addEventListener("resize", () => {
       if (window.innerWidth <= 959) this.cardBlock = true
       else this.cardBlock = false
-    })
+    }),
+    this.getOrderHistory()
+  },
+
+  methods: {
+    goToDetail() {
```

Comment:
A lot of this is not inline with the engineering handbook, please separate each method with spaces.

```
methods: {
  goToDetail(){
    // Code
  }

  async getOrderHistory(){
    // Code
  }
}
```